ql/src/java/org/apache/hadoop/hive/ql/exec/mr/ExecMapper.java

```diff
@@ -20,10 +20,12 @@
 
 import java.io.IOException;
 import java.net.URLClassLoader;
+import java.util.ArrayList;
```

Comment:
need to bump up the pom build versions too ?

Reply:
Ack. Version bump up is required. Will push the changes